src/libraries/Microsoft.Extensions.Logging.Console/src/ConsoleLoggerFactoryExtensions.cs

```diff
@@ -42,5 +50,123 @@ public static ILoggingBuilder AddConsole(this ILoggingBuilder builder, Action<Co
 
             return builder;
         }
+
+        /// <summary>
+        /// Add and configure a console log formatter named 'json' to the factory.
+        /// </summary>
+        /// <param name="builder">The <see cref="ILoggingBuilder"/> to use.</param>
+        /// <param name="configure">A delegate to configure the <see cref="ConsoleLogger"/> options for the built-in default log formatter.</param>
+        public static ILoggingBuilder AddDefaultConsoleLogFormatter(this ILoggingBuilder builder, Action<DefaultConsoleLogFormatterOptions> configure)
```

Comment:
>AddDefaultConsoleLogFormatter [](start = 38, length = 29)

The code in this methods is almost exact as the code in AddSystemdConsoleLogFormatter and AddJsonConsoleLogFormatter. can we just refactor the code into one internal method and let the public methods just call the internal method?